app/src/main/res/values-es/strings.xml

```diff
@@ -17,4 +17,13 @@
     <string name="reg_vm_valid_email">El Email está siendo utilizado en otra cuenta</string>
 
     <string name="hello_blank_fragment">Hola Fragmento en Blanco</string>
+    <!-- Google Cliente Web -->
+    <string name="default_web_cliente_id">11042396908-c4tf4cmfen29e6oln7uum134r3vsvo08.apps.googleusercontent.com</string>
```

Comment:
@jmacboy About this, we have to use any string and update on our local machines right?

Reply:
yes, that should be done

Reply:
Done!